### DISCUSSION TITLE
Como acceder a últimos 7 días

### DISCUSSION THREAD

**Discussion by jmgarcia2** [Paco8/plugin.video.movistarplus#13] [General]
Hola Paco8,
¡Magnífico trabajo!.

Es la primera vez que accedo a tu trabajo.. y todavía estoy absorbiendo y descubriendo
sus facilidades, pero hay una a la que no encuentro en modo de acceder, y es a **Últimos 7 días**.

Me podías facilitar una guía de burros :-) sobre como acceder y como usarla.

Muchas gracias,

**Comment by Paco8** [comment]
No está implementado todavía.

**Reply by jmgarcia2** [reply]
Gracias.
